@here/harp-mapview/lib/ScreenProjector.ts

```diff
@@ -92,6 +92,37 @@ export class ScreenProjector {
         return undefined;
     }
 
+    /**
+     * Test if the area around the specified point is visible on the screen.
+     *
+     * @param {(Vector3Like)} source The centered source vector to project.
+     * @param {(Number)} width Half of the width of the area in NDC space [0..1].
+     * @param {(Number)} height Half of the height of the area in NDC space [0..1].
+     * @param {THREE.Vector2} target The target vector.
+     * @returns {THREE.Vector2} The projected vector (the parameter 'target') or undefined if
+     * the area is completely outside the screen.
+     */
+    projectAreaToScreen(
+        source: Vector3Like,
+        width: number,
+        height: number,
+        target: THREE.Vector2 = new THREE.Vector2()
+    ): THREE.Vector2 | undefined {
+        width *= 2;
+        height *= 2;
```

Comment:
why multiply by 2? If p is at the center, the area right side is at p.x + halfWidth.